libpod/runtime_img.go

```diff
@@ -295,27 +294,19 @@ func (r *Runtime) LoadImage(ctx context.Context, name, inputFile string, writer
 			return dockerarchive.ParseReference(inputFile)
 		},
 		func() (types.ImageReference, error) {
-			return ociarchive.NewReference(inputFile, name) // name may be ""
-		},
-		func() (types.ImageReference, error) {
-			// prepend "localhost/" to support local image saved with this semantics
-			if !strings.Contains(name, "/") {
-				return ociarchive.NewReference(inputFile, fmt.Sprintf("%s/%s", image.DefaultLocalRegistry, name))
+			if name != "" {
+				return nil, errors.Errorf("oci format does not support additional name %s", name)
```

Comment:
BTW this will never be seen by users, the way `err` is currently maintained by the loop. So the UI might need a bit more thought (or perhaps just give up and return `nil, nil` here — probably not, but that does depend on what the refactoring in callers ends up doing).